hwilib/commands.py

```diff
@@ -180,9 +202,9 @@ def getdescriptors(client, account=0):
 
     for internal in [False, True]:
         descriptors = []
-        desc1 = getdescriptor(client, master_fpr=master_fpr, testnet=client.is_testnet, internal=internal, sh_wpkh=False, wpkh=False, account=account)
```

Comment:
`master_xpub` is an unrelated change. You may need to rebase.

Reply:
should be fixed, was an artifact of rebasing incorrectly onto master